app/cdap/components/PipelineDetails/ProfilesListView/ProfileCustomizePopover/ProfileCustomizeContent/index.js

```diff
@@ -23,12 +23,13 @@ import Accordion, {
   AccordionPane,
 } from 'components/shared/Accordion';
 import { MyCloudApi } from 'api/cloud';
-import uuidV4 from 'uuid/v4';
 import { extractProfileName } from 'components/Cloud/Profiles/Store/ActionCreator';
 import IconSVG from 'components/shared/IconSVG';
 import cloneDeep from 'lodash/cloneDeep';
 import classnames from 'classnames';
-import { WrappedWidgetWrapper } from 'components/shared/ConfigurationGroup/WidgetWrapper';
+import ProfilePropertyRow from 'components/PipelineDetails/ProfilesListView/ProfileCustomizePopover/ProfilePropertyRow';
```

Comment:
would this be better `import ProfilePropertyRow from '../ProfilePropertyRow`? I think one or two steps its nice to just use the . or ../ @njbriggs how do you feel about imports using `.` or `../`?